drivers/bus/pci/pdo.c

```diff
@@ -1394,12 +1397,14 @@ PdoStartDevice(
         /* OR with the previous value */
         Command |= DeviceExtension->PciDevice->PciConfig.Command;
 
-        HalSetBusDataByOffset(PCIConfiguration,
-                              DeviceExtension->PciDevice->BusNumber,
-                              DeviceExtension->PciDevice->SlotNumber.u.AsULONG,
-                              &Command,
-                              FIELD_OFFSET(PCI_COMMON_CONFIG, Command),
-                              sizeof(USHORT));
+        Size = HalSetBusDataByOffset(PCIConfiguration,
+                                     DeviceExtension->PciDevice->BusNumber,
+                                     DeviceExtension->PciDevice->SlotNumber.u.AsULONG,
+                                     &Command,
+                                     FIELD_OFFSET(PCI_COMMON_CONFIG, Command),
+                                     sizeof(USHORT));
+        if (Size == 0)
```

Comment:
I wonder whether a comparison wrt. the expected data size to be written would be better: `if (Size != sizeof(USHORT))`